sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/TestEnvironment.cs

```diff
@@ -35,24 +35,40 @@ public static class TestEnvironment
         private static readonly Lazy<string> EventHubsSecretInstance =
             new Lazy<string>(() => ReadAndVerifyEnvironmentVariable("EVENT_HUBS_SECRET"), LazyThreadSafetyMode.PublicationOnly);
 
+        /// <summary>The environment variable value for the namespace connection string, lazily evaluated.</summary>
+        private static readonly Lazy<string> EventHubsNamespaceConnectionString =
+            new Lazy<string>(() => ReadConnectionStringFromEnvironment("EVENT_HUBS_NAMESPACE_CONNECTION_STRING"), LazyThreadSafetyMode.PublicationOnly);
+
+        /// <summary>The environment variable value for the event hub name, lazily evaluated.</summary>
+        private static readonly Lazy<string> EventHubsEventHubName =
+            new Lazy<string>(() => ReadEnvironmentVariable("EVENT_HUBS_EVENT_HUB_NAME"), LazyThreadSafetyMode.PublicationOnly);
+
```

Comment:
Was thinking about the names earlier and considering that it may be a good idea to tag these as "overrides" to make sure that its clear you shouldn't use them unless you know what you're doing, because it could cause test failures.   Maybe a pattern like `EVENT_HUBS_OVERRIDE_NAMESPACE_CONNECTION_STRING`  and `EVENT_HUBS_OVERRIDE_EVENT_HUB_NAME`?      

Does that sound reasonable to you or am I over thinking things?

Reply:
Nice touches